### DISCUSSION TITLE
How to subscribe to Multi Level

### DISCUSSION THREAD

**Discussion by carrisian** [dotnet/MQTTnet#1485] [Q&A]
I have several topic for example:
/server/identify
/sever/control
/server/take/object
I want to subscribe to everything on the level server

I try to execute this code and it doesn't work.

_private async Task mqttClient_ConnectedAsync(MqttClientConnectedEventArgs arg)
        {      
            await mqttClient.SubscribeAsync(new MqttTopicFilterBuilder().WithTopic("/server/#").Build());
        }_

What's wrong? Thank you in advanced

**Comment by chkr1011** [comment]
Topics should not start with a /. Please try without the leading /.
